server.go

```diff
@@ -119,8 +119,15 @@ type Server struct {
 	traceBackend *internalTraceBackend
 }
 
-// NewFromConfig creates a new veneur server from a configuration specification.
-func NewFromConfig(conf Config) (*Server, error) {
+// SetLogger sets the default logger in veneur to the passed value.
+func SetLogger(logger *logrus.Logger) {
+	log = logger
```

Comment:
This doesn't trip the race detector? (We clearly don't use it in a way that races, but…)

Reply:
We don't use that function in tests! Only main.go (: